js/about/preferences.js

```diff
@@ -576,6 +672,7 @@ class PaymentsTab extends ImmutableComponent {
                 {this.btcToCurrencyString(this.props.ledgerData.get('balance'))}
                 </span>
                 {this.walletButton}
+                {this.paymentHistoryButton}
```

Comment:
fixed in: https://github.com/brave/browser-laptop/pull/3391/commits/6c8274d4abb47a2063d4d24c55d6369083f7a89f#diff-e3eeb751016b2ce9f8278efce585a461R581